Generator/Generator/MethodGen.swift

```diff
@@ -754,7 +753,7 @@ func generateMethod(_ p: Printer, method: MethodDefinition, className: String, c
     
     p ("\(declarationTokens)(\(argumentsList))\(returnClause)") {
         if staticAttribute == nil {
-            p("if handle == nil { Wrapped.attemptToUseObjectFreedByGodot() }")
+            p("assertValidity()")
         }
```

Comment:
The point of using this long-name was to spot it easily on crash reports that we are getting from users, it cuts down on the detective work, please do not rename it.